simplenote_cli/sncli.py

```diff
@@ -1192,30 +1194,32 @@ def main(argv):
             key = arg
         elif opt in [ '-t', '--title']:
             title = arg
+        elif opt in [ '-c', '--config']:
+            config = arg
         else:
             print u'ERROR: Unhandled option'
             usage()
 
     if not args:
-        sncli(sync).gui(key)
+        sncli(sync, verbose, config).gui(key)
```

Comment:
It was all a bit of reshuffling to easier accomodate adding the `config` variable. `sncli_start` (line 1207) now has default variables that equal the relevant option (`sync=sync, ...`), and the call to `sncli` has verbose there to look neater (it could alternatively be `sncli(sync, config=config).gui(key)`, but no difference).